### ISSUE TITLE
Biosuit Doesn't Protect Against Smoke

### ISSUE THREAD

**Issue by KasparoVy** [ParadiseSS13/Paradise#11516]
**Problem Description**:
What is the problem?
The MOPP suit you can wear from medical Level 3 lockers doesn't protect against smoke, namely blobsmoke.

**What did you expect to happen**: 
It raises into question the point of the suit's existence- I expected a degree of protection from a suit supposed to protect from shenanigans

**What happened instead**:
Blobsmoke wastes you regardless of the biohazard protection you're wearing.

**Why is this bad/What are the consequences:**
You just gotta facetank the blob I guess and have no counter to smoke aside from being somewhere else.

**Steps to reproduce the problem**:
Wear biosuit, fight blob.

**Possibly related stuff (which gamemode was it? What were you doing at the time? Was anything else out of the ordinary happening?)**:
Blob gamemode.

**Comment by Kyep** [comment]
AFAIK nothing counters blobsmoke.
You can be wearing a full hardsuit, with internals and all, and blob smoke will still burn you right through your airtight, sealed hardsuit.

**Comment by KasparoVy** [comment]
If that's intended behaviour this can be closed

**Comment by Kyep** [comment]
I am unsure if it was intended or not.
Even if it was intended by the original author, that doesn't necessarily mean we want to keep it that way.

**Comment by The-Driker** [comment]
If the suit is suppose to protect you from smoke hazards in a universe that knows about blobs, I'd say it's probably not intended behavior.

**Comment by Couls** [comment]
It’s the way the smoke is handled, works like a gas that you inhale. The suit won’t save you from it but internals should

**Comment by Allfd** [comment]
Yeah, there are a few effects like this that don't make a ton of sense.  Like how how certain poison gasses or effects can take out people with nobreath, but N2O won't knock them out for surgery.
